docs/references/ic-interface-spec.md

```diff
@@ -1158,7 +1158,7 @@ When canisters explicitly reject a message (see [Public methods](#system-api-req
 
 ### Error codes {#error-codes}
 
-Implementations of the API can provide additional details for rejected messages in the form of a textual label identifying the error condition. API clients can use these labels to handle errors programmatically or suggest recovery paths to the user. The specification reserves error codes matching the regular expression `IC[0-9]+` (e.g., `IC502`) for the DFINITY implementation of the API.
+Implementations of the HTTPS and System APIs can provide additional details for rejected messages in the form of a textual label identifying the error condition. API clients and canisters can use these labels to handle errors programmatically or suggest recovery paths to the user. In the HTTPS API, the specification reserves error codes matching the regular expression `IC[0-9]+` (e.g., `IC502`) for the DFINITY implementation of the API.
```

Comment:
Is this accurate given that the proposed system API would return only a numerical value? Unless I missed something you are proposing that instead of returning `IC502`, the system API would return `502`.

Reply:
Oops. I rephrased this now to specifically say that labels are only used in the HTTPS API. Thanks!